### DISCUSSION TITLE
HAR method is broken on dungeon simulator

### DISCUSSION THREAD

**Discussion by Anjajaja** [HafisCZ/sf-tools#60] [General]
Hey! As of recently (I noticed today) the HAR method is not working anymore for me and my friend. When importing a HAR file, it just does not fill out anything anymore.
We are using the new account system, could that be the reason? Does anyone elste have this Issue?

**Comment by HafisCZ** [comment]
It seems to be working fine for me. Are you sure you created the capture properly?
